devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy

```diff
@@ -82,6 +82,9 @@ metadata {
 		// Third Reality
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0019", manufacturer: "Third Reality, Inc", model: "3RSS008Z", deviceJoinName: "RealitySwitch Plus"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0019", manufacturer: "Third Reality, Inc", model: "3RSS007Z", deviceJoinName: "RealitySwitch"
+		
+		// EZEX
```

Comment:
@juhaki please move this section up so it's in alphabetical order by manufacturer

Reply:
Ok. I will patch to this about alphabetical order move this section.